QRCoder/Base64QRCode.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿#if !NETSTANDARD1_1
```

Comment:
Thinking about this a bit more, I think this might affect NET45 targets (which is included in NS1.1). May need to add a NET45 target explicitly to the csproj to avoid this. Will take a look once I have access to my dev machine.